### ISSUE TITLE
How to hide selection that isn't active?

### ISSUE THREAD

**Issue by flatcapco** [bgrins/spectrum#267]
I'm not sure if this is an issue or poor understanding by me:
When I use:

   showSelectionPalette: true,
   maxSelectionSize: 1,

It adds another palette colour after my original palette which is great.. however if I click back to one of the original palette colours it still shows the previous selection after the original palette list.

How do I hide the newly created colour (the one that wasn't in the original palette) if it is no longer active?

In simple terms - if its not active or in the original palette I don't want to see it.

Many thanks for such a great plugin
